src/log.cpp

```diff
@@ -432,9 +434,16 @@ static void print_precise_timestamp(std::ostream& out) noexcept
 	} catch(...) {}
 }
 
+void set_log_sanitize(bool sanitize) {
+	log_sanitization = sanitize;
+}
+
 std::string sanitize_log(const std::string& logstr)
 {
 	std::string str = logstr;
+	if (!log_sanitization) {
```

Comment:
Might as well put the condition first in the function.

Reply:
Moved.